src/node/entities.h

```diff
@@ -85,6 +85,7 @@ namespace ccf
     static constexpr auto USER_CODE_IDS = "ccf.users.code_ids";
     static constexpr auto CONFIGURATION = "ccf.config";
     static constexpr auto SUBMITTED_SHARES = "ccf.submitted_shares";
+    static constexpr auto SNAPSHOT_EVIDENCES = "ccf.snapshot_evidences";
```

Comment:
I believe evidence is nearly always uncountable, and so this should be SNAPSHOT_EVIDENCE.

Reply:
You're right - I have singular-ised the table name.